asyn/Makefile

```diff
@@ -354,3 +355,9 @@ else
 	@$(MV) $(notdir $@) $@
 endif
 
+$(LIB_PREFIX)asyn$(LIB_SUFFIX): USR_CPPFLAGS += -DBUILDING_asyn_API
+ifeq ($(SHARED_LIBRARIES),YES)
+  $(SHRLIB_PREFIX)asyn$(SHRLIB_SUFFIX): USR_CPPFLAGS += -DBUILDING_asyn_API
+endif
```

Comment:
Since libasyn is the only output from this directory, this could be simplified as:

```
USR_CPPFLAGS += -DBUILDING_asyn_API
```